tests/integration/test_github.py

```diff
@@ -36,6 +36,18 @@ def test_authorize(self):
 
         assert isinstance(auth, github3.auths.Authorization)
 
```

Comment:
@itsmemattchung so I got rid of `add_email_address` (singular) and `delete_email_address` (singular) because I realized that it made little sense that we special cased the removal of a single address for a user when the difference is `[]`.

We absolutely do want to add a test for the case where someone calls `add_email_addresses` with an empty list and `self.session.post.called is False`. Great catch!